tools/make.py

```diff
@@ -173,6 +186,11 @@
                       default=False,
                       help="Link with mbed test library")
 
+    parser.add_argument("--metadata",
```

Comment:
probably just some milliseconds, perhaps ten - I think - this seems to just pick pre-defined properties all over the place and put them all to one place - json file.